<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clicker</class>
 <widget class="QWidget" name="Clicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <widget class="QFrame" name="fr_controls">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>71</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLCDNumber" name="lcd_ellipsed_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>217</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbB2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbB4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>106</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbShift">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>162</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCtrl">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>139</y>
      <width>48</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbB4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>106</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbB1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbOnOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbB3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>83</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbShift">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>162</y>
      <width>48</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbB1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>37</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCtrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbB2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDongle">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>48</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>193</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbB3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>83</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>55</width>
     <height>233</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
